clj-refactor.el

```diff
@@ -2484,6 +2488,7 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-promote-function
                             "line" line
                             "column" column
                             "name" symbol
+                            "ignore-paths" cljr-middleware-ignore-paths
```

Comment:
I wonder if `ignored-paths` won't be a better name for this, as normally I'd assume something named `ignore-paths` means something else.